server/accounts.go

```diff
@@ -1414,7 +1430,7 @@ func (a *Account) streamActivationExpired(subject string) {
 }
 
 // These are import service specific versions for when an activation expires.
-func (a *Account) serviceActivationExpired(subject string) {
+func (a *Account) serviceActivationExpired(exportAcc *Account, subject string) {
```

Comment:
You pass exportAcc to this function but did not change the map lookup to an array scan..

Reply:
Yes matched the function signature but services still use the map and direct lookup.

Reply:
Oh yes, this is "service" and you changed only "streams"... but I would argue that passing `exportAcc` and not using it in this function is a bit confusing..

Reply:
ok removed.